Composer/packages/client/src/pages/home/Home.tsx

```diff
@@ -158,114 +144,105 @@ const Home: React.FC<RouteComponentProps> = () => {
   ];
   return (
     <div css={home.outline}>
-      <Toolbar toolbarItems={toolbarItems} />
+      <h1 css={home.title}>{formatMessage(`Bot Framework Composer`)}</h1>
```

Comment:
does this need to be localized?

Reply:
It keeps same to current main, should we change it ?

Reply:
I guess the name of the app should be the same across all locales.